src/App.tsx

```diff
@@ -46,6 +46,7 @@ function App() {
 
         do {
             result = await sdk.playlists.getPlaylistItems(match[1], 'DE', 'offset,limit,next,items(track(id,name,artists(name),album(release_date)))', limit, offset);
+            result.items = result.items.filter(item => item.track?.name);
```

Comment:
Could you either store the items in a seperate varibale und apply the filter to this + update line 49 accordingly or inline the filtering directly in line 49? Seems more elegant to me!

Reply:
Good point, is fixed now